LondonBritishLibrary/orient/BLorient788.xml

```diff
@@ -142,9 +142,13 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                             <list>
                                 <item xml:id="a1">
                                     <locus target="#165r"/>
-                                    <desc type="OwnershipNote"></desc>
-                                    <q xml:lang="gez">ዝመጽሐፍ፡ ዘቅዱስ፡ ሩፋኤል፡ ፈውስ፡ መንፈሳዊ፤ ዘምስለ፡ ቀንዲል፤ ወሥርዓተ፡ ቤተ፡ ክርስቲያን። <foreign xml:lang="am">የፍትሐት፡ መጽሐፍ፤ ምስለ፡ ተክሊል፤ የሌሊትና፡
-                                        የመዓልት፡ ሰዓታት፤ ምስለ፡ ጸሎተ፡ ዕጣን፤ ያዓመት፡ እግዜር፡ ነግሥ፤ ተአምረ፡ እግዝእትነ፡ ማርያም፤</foreign> እሉ፡ ኵሎሙ፡ ዘወሀቦሙ፡ <persName ref=""><roleName>መምህሬ፡</roleName> ኢሳይያስ፡</persName> ገብሩ፡ ለሩፋኤል። 
+                                    <desc type="OwnershipNote">Note written by an apparently later hand stating that this manuscript belongs
+                                        to a church of Rufāʾel, possibly <placeName ref="INS0956Rufael"/>, to which it was donated with a group of
```

Comment:
I would put it
`a church of <persName ref="PRS8083Raphael" type="tabot">Rufāʾel</persName>` (since we do not know exactly which) - and I could not find INS0956Rufael - no PR yet?

Reply:
https://github.com/BetaMasaheft/Institutions/pull/21
:)